webosose/ares-cli
# Add error handling in ares-device and ares-log

:Release Notes:
Add error handling in ares-device and ares-log

:Detailed Notes:
- Add error handling for resource-monitor option of ares-device
- Add error handling for save option of ares-log
- Change file name format to including "_" in timestamp

:Testing Performed:
1. Pass unit test
2. Pass eslint
3. Check error messages below test cases
 - ares-device.js -r -t 0
   > ares-device ERR! [Tips]: Please use nonnegative integer value for the "time-interval" option
 - ares-device.js -r -t 1 -s invalidDir/aaa
   > ares-device ERR! [syscall failure]: ENOENT: ...
     ares-device ERR! [Tips]: Please check if the path is valid ...
 - ares-log.js -n 10 -s invalidDir/aaa
   > ares-log ERR! [syscall failure]: ENOENT: ...
     ares-log ERR! [Tips]: Please check if the path is valid ...
4. Check file name below test cases
 - ares-device -c
 - ares-device -r -s

:Issues Addressed:
[WRN-14542] Add error handling for ares-device and ares-log
